src/main/java/org/folio/circulation/domain/LoanRepository.java

```diff
@@ -52,6 +52,7 @@
   private static final Logger log = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
   private static final String ITEM_STATUS = "itemStatus";
   private static final String ITEM_ID = "itemId";
+  private static final int LOANS_LIMIT = 100;
```

Comment:
Why limit=100 was introduced? I suppose all user loans are needed.

Reply:
fixed, now it's the max integer value